src/components/experimental/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+import { IconProps } from '../../../icons';
+import { getSemanticValue } from '../../../essentials/experimental';
+
+export interface IconButtonProps {
+    disabled?: boolean;
+    active?: boolean;
+    variant?: 'standard' | 'tonal';
+    Icon: React.FC<IconProps>;
+    onClick: () => void;
+}
+
+const StandardIconContainer = styled.button<Omit<IconButtonProps, 'Icon' | 'onClick'>>`
```

Comment:
In experimental components we usually base the component on the component from 'react-aria-components'. Have a look at the Button as an example

Reply:
Done! ✅